hedera-mirror-rest/api/v1/openapi.yml

```
@@ -3337,6 +3337,26 @@ components:
           type: string
         name:
           $ref: "#/components/schemas/TransactionTypes"
+        nft_transfers:
+          type: array
+          items:
+            type: object
+            properties:
+              is_approval:
+                type: boolean
+              receiver_account_id:
+                $ref: "#/components/schemas/EntityId"
+              sender_account_id:
+                $ref: "#/components/schemas/EntityId"
+              serial_number:
+                example: 1
+                format: int64
+                type: integer
+              token_id:
+                $ref: "#/components/schemas/EntityId"
+            required:
+              - token_id
+              - serial_number
```

every field is required
```suggestion
              - is_approval
              - receiver_account_id
              - sender_account_id
              - token_id
              - serial_number
```